docker/config-modifier/account-lookup-service.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  DATABASE: {
+    HOST: 'mysql-als',
+    SCHEMA: 'account_lookup'
+  },
+  SWITCH_ENDPOINT: 'http://central-ledger:3001',
+  ALS_GENERAL_CACHE_CONFIG: {
```

Comment:
Why is it ALS_GENERAL_CACHE_CONFIG here where as you renamed that in default.json to GENERAL_CACHE_CONFIG?

Reply:
Good catch...